src/components/Level/staticLevels/index.js

```diff
@@ -0,0 +1,213 @@
+import { ReactComponent as OnePointRedDino } from '../../../icons/dinosaurs/red-one-point.svg';
+import { ReactComponent as TwoPointsRedDino } from '../../../icons/dinosaurs/red-two-points.svg';
+import { ReactComponent as ThreePointsRedDino } from '../../../icons/dinosaurs/red-three-points.svg';
+import { ReactComponent as FivePointsRedDino } from '../../../icons/dinosaurs/red-five-points.svg';
+import { ReactComponent as TenPointsRedDino } from '../../../icons/dinosaurs/red-ten-points.svg';
+
+import { ReactComponent as OnePointBlueDino } from '../../../icons/dinosaurs/blue-one-point.svg';
+import { ReactComponent as TwoPointsBlueDino } from '../../../icons/dinosaurs/blue-two-points.svg';
+import { ReactComponent as ThreePointsBlueDino } from '../../../icons/dinosaurs/blue-three-points.svg';
+import { ReactComponent as FivePointsBlueDino } from '../../../icons/dinosaurs/blue-five-points.svg';
+import { ReactComponent as TenPointsBlueDino } from '../../../icons/dinosaurs/blue-ten-points.svg';
+
+export const dinos = {
```

Comment:
I think this file/folder makes more sense somewhere like `src/data` instead of `src/components/Level/staticLevels`. Also, what do you think about separating out all of the different data into different files? Like `src/data/dinos.js`, `src/data/legends.js` and so on?

Reply:
I agree src/data makes more sense as other components will use this data. I can change that.

Potential issue that I am looking for suggestions on. With the current setup legends only displays 3 dinos per legend board. I did this purposely since that is how beginner difficulty will work. I am open to separating the files out but I need some help figuring out how we are going to add in more dinos under legend if the difficulty is Average. And for Hard adding in the skeleton dinos for subtraction. Kind of stumped on what we will do for that.
The links for the level menu I just created will bring you to #/levels/Beginner/:1 for example.

Reply:
```js
17: {
  7: 2,
  9: 5,
  10: 10,
  11: 20,
},
```
You can just create a new legend with four dinos instead of 3 and reference that. If we want to separate it out by difficulty, we can add nested a nested hash like
```js
const levels = {
  beginner: {
    1: {
      // data
    }
  },
  hard: {
    1: {
      // data
    }
  }
}
```

Reply:
done. added both hard and average difficulty. 10 levels each.